cmd/shipper/main.go

```diff
@@ -1,16 +1,106 @@
 package main
 
 import (
+	"gz-beat-shipper/configs"
```

Comment:
Question about Go, is there any convention regarding import orders? For example, first standard library, then other packages, then other parts of the own package?

Reply:
I didn't find any convention for importing orders. I just see this commit as an example: https://github.com/golang/tools/commit/ed69e84b1518b5857a9f4e01d1f9cefdcc45246e 

Anyway, this is automatically ordered by the IDE using the linter that Go has per se.

Reply:
Convention is to order them in alphabetical order. Formatters will take care of that.